### PR TITLE
lv_img_conv: Remove

### LINKED ISSUE #318122: Build failure: lv_img_conv
### Steps To Reproduce

1. `nix run nixpkgs#lv_img_conv` w/ `nixos-24.05` channel
2. or check hydra: https://hydra.nixos.org/build/260862301

### Build log

```
lv_img_conv> npm ERR! In file included from ../../nan/nan.h:178,
lv_img_conv> npm ERR!                  from ../src/lmdb-js.h:32,
lv_img_conv> npm ERR!                  from ../src/lmdb-js.cpp:1:
lv_img_conv> npm ERR! ../../nan/nan_callbacks.h:55:23: error: 'AccessorSignature' is not a member of 'v8'
lv_img_conv> npm ERR!    55 | typedef v8::Local<v8::AccessorSignature> Sig;
lv_img_conv> npm ERR!       |                       ^~~~~~~~~~~~~~~~~
lv_img_conv> npm ERR! ../../nan/nan_callbacks.h:55:40: error: template argument 1 is invalid
lv_img_conv> npm ERR!    55 | typedef v8::Local<v8::AccessorSignature> Sig;
lv_img_conv> npm ERR!       |                                        ^
lv_img_conv> npm ERR! ../../nan/nan.h: In function 'void Nan::SetAccessor(v8::Local<v8::ObjectTemplate>, v8::Local<v8::String>, GetterCallback, SetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute, imp::Sig)':
lv_img_conv> npm ERR! ../../nan/nan.h:2544:19: error: no matching function for call to 'v8::ObjectTemplate::SetAccessor(v8::Local<v8::String>&, void (*&)(v8::Local<v8::Name>, const v8::PropertyCallbackInfo<v8::Value>&), void (*&)(v8::Local<v8::Name>, v8::Local<v8::Value>, const v8::PropertyCallbackInfo<void>&), v8::Local<v8::Object>&, v8::AccessControl&, v8::PropertyAttribute&, Nan::imp::Sig&)'
lv_img_conv> npm ERR!  2544 |   tpl->SetAccessor(
lv_img_conv> npm ERR!       |   ~~~~~~~~~~~~~~~~^
lv_img_conv> npm ERR!  2545 |       name
lv_img_conv> npm ERR!       |       ~~~~
lv_img_conv> npm ERR!  2546 |     , getter_
lv_img_conv> npm ERR!       |     ~~~~~~~~~
lv_img_conv> npm ERR!  2547 |     , setter_
lv_img_conv> npm ERR!       |     ~~~~~~~~~
lv_img_conv> npm ERR!  2548 |     , obj
lv_img_conv> npm ERR!       |     ~~~~~
lv_img_conv> npm ERR!  2549 |     , settings
lv_img_conv> npm ERR!       |     ~~~~~~~~~~
lv_img_conv> npm ERR!  2550 |     , attribute
lv_img_conv> npm ERR!       |     ~~~~~~~~~~~
lv_img_conv> npm ERR!  2551 |     , signature);
lv_img_conv> npm ERR!       |     ~~~~~~~~~~~~
lv_img_conv> npm ERR! In file included from /nix/store/nxbymm4w70fpp56jn7wqyrjiyd0zlhq4-nodejs-20.12.2-source/deps/v8/include/v8-function.h:15,
lv_img_conv> npm ERR!                  from /nix/store/nxbymm4w70fpp56jn7wqyrjiyd0zlhq4-nodejs-20.12.2-source/deps/v8/include/v8.h:33,
lv_img_conv> npm ERR!                  from ../src/lmdb-js.h:29:
lv_img_conv> npm ERR! /nix/store/nxbymm4w70fpp56jn7wqyrjiyd0zlhq4-nodejs-20.12.2-source/deps/v8/include/v8-template.h:809:8: note: candidate: 'void v8::ObjectTemplate::SetAccessor(v8::Local<v8::String>, v8::AccessorGetterCallback, v8::AccessorSetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute, v8::SideEffectType, v8::SideEffectType)'
lv_img_conv> npm ERR!   809 |   void SetAccessor(
lv_img_conv> npm ERR!       |        ^~~~~~~~~~~
lv_img_conv> npm ERR! /nix/store/nxbymm4w70fpp56jn7wqyrjiyd0zlhq4-nodejs-20.12.2-source/deps/v8/include/v8-template.h:814:22: note:   no known conversion for argument 7 from 'Nan::imp::Sig' {aka 'int'} to 'v8::SideEffectType'
lv_img_conv> npm ERR!   814 |       SideEffectType getter_side_effect_type = SideEffectType::kHasSideEffect,
lv_img_conv> npm ERR!       |       ~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lv_img_conv> npm ERR! /nix/store/nxbymm4w70fpp56jn7wqyrjiyd0zlhq4-nodejs-20.12.2-source/deps/v8/include/v8-template.h:816:8: note: candidate: 'void v8::ObjectTemplate::SetAccessor(v8::Local<v8::Name>, v8::AccessorNameGetterCallback, v8::AccessorNameSetterCallback, v8::Local<v8::Value>, v8::AccessControl, v8::PropertyAttribute, v8::SideEffectType, v8::SideEffectType)'
lv_img_conv> npm ERR!   816 |   void SetAccessor(
lv_img_conv> npm ERR!       |        ^~~~~~~~~~~
lv_img_conv> npm ERR! /nix/store/nxbymm4w70fpp56jn7wqyrjiyd0zlhq4-nodejs-20.12.2-source/deps/v8/include/v8-template.h:821:22: note:   no known conversion for argument 7 from 'Nan::imp::Sig' {aka 'int'} to 'v8::SideEffectType'
lv_img_conv> npm ERR!   821 |       SideEffectType getter_side_effect_type = SideEffectType::kHasSideEffect,
lv_img_conv> npm ERR!       |       ~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
lv_img_conv> npm ERR! make: *** [lmdb.target.mk:140: Release/obj.target/lmdb/src/lmdb-js.o] Error 1
lv_img_conv> npm ERR! gyp ERR! build error
lv_img_conv> npm ERR! gyp ERR! stack Error: `make` failed with exit code: 2

```

### Notify maintainers

@StarGate01 ping :)

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[rrix@virtuous-cassette:~/Code/InfiniTime]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.32, NixOS, 24.05 (Uakari), 24.05pre-git`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.2`
 - nixpkgs: `/home/rrix/Code/nixpkgs`
```

---

### PR DESCRIPTION
**Pull request by StarGate01**
## Description of changes

Closes #318122 . `lv_img_conv` fails to build on the current Node version.

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/development/tools/lv_img_conv/default.nix [hunk 1]
```diff
@@ -60,5 +60,6 @@ buildNpmPackage rec {
     homepage = "https://github.com/lvgl/lv_img_conv";
     license = licenses.mit;
     maintainers = with maintainers; [ stargate01 ];
+    broken = true;
```
Comment (Aleksanaa) [inline pkgs/development/tools/lv_img_conv/default.nix:63]:
```suggestion
    # Added 2024-06-18, https://github.com/NixOS/nixpkgs/issues/318122
    broken = true;
```

**Comment by SuperSandro2000** [comment]
Why not remove it entirely? Doesn't seem to be needed anymore

**Comment by StarGate01** [comment]
@SuperSandro2000 Fair point, I wanted to conserve the package for the unlikely case that someone still used it. But since it does not even compile anymore, removing it is the better option. I updated the PR.

**Review by pluiedev — CHANGES_REQUESTED** [review]
Probably need to add an entry to `aliases.nix` indicating that it has been removed

**Review by Aleksanaa — APPROVED** [review]

**Pull request merged**